File: Command.java
import java.lang.*;

class Command
{
    public static void main(String Arg[])
    {
        System.out.println("Number of command line arguments are : "+Arg.length);

        System.out.println("First number is : "+Arg[0]);
        System.out.println("First number is : "+Arg[1]);

        int ans = Integer.parseInt(Arg[0]) + Integer.parseInt(Arg[1]);
        System.out.println("Addition is :"+ans);
    }
}

// Note : Error whatsapp karanyaadhi khalacha wacha

// javac Command.java
// java Command 10 11

File: Command.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char *argv[])
{
    int ans = 0;
    printf("Name of executable is : %s\n",argv[0]);

    printf("Number of command line arguments are : %d\n",argc);

    ans = atoi(argv[1]) + atoi(argv[2]);
    printf("Addition is : %d\n",ans);

    return 0;
}

// Note : Error whatsapp karanyaadhi khalacha wacha

// gcc Command.c -o Myexe
// ./Myexe 11 21